### DISCUSSION TITLE
The `/var/lib/marzban` is empty

### DISCUSSION THREAD

**Discussion by hamidmayeli** [Gozargah/Marzban#1203] [Q&A] [answered]
I made a copy of the `docker-compose` file to ran that on my local machine to play around with it and see what features it provides. I have created a few users but there isn't any actual use on the server. To my surprise the `/var/lib/marzban` is empty and restarting the container results in data loss.

**Comment by M03ED** [comment, marked as answer]
That's because you didn't set SQLALCHEMY_DATABASE_URL variable in env file , also there is lot's of usage for this folder 
And if you want to change files and test them I suggest you run marzban manually
